soccer/src/soccer/strategy/agent/position/seeking.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include <cmath>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+#include <spdlog/spdlog.h>
+
+#include <rj_msgs/action/robot_move.hpp>
+
+#include "planning/instant.hpp"
+#include "position.hpp"
+#include "rj_common/field_dimensions.hpp"
+#include "rj_common/time.hpp"
+#include "rj_constants/constants.hpp"
+#include "rj_geometry/geometry_conversions.hpp"
+#include "rj_geometry/point.hpp"
+#include "role_interface.hpp"
+
+namespace strategy {
+
+/*
+ * The Seeking role provides the implementation for a offensive robot that
+ * is trying to get open, so that they can receive a pass
+ */
+class Seeking : public RoleInterface {
+public:
```

Comment:
see the GH action warning: explicitly `=default` the other four required ctors/operators see "rule of five" section here https://en.cppreference.com/w/cpp/language/rule_of_three or ask me for more info